go.mod

```diff
@@ -55,3 +55,75 @@ require (
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/spf13/cast v1.4.0
 )
+
+require (
```

Comment:
why did we double the require, can you run go mod tidy ?

Reply:
I already ran go mod tidy and that is the result of this. Thanks